components/DateSelector/DateSelector.jsx

```diff
@@ -25,6 +25,7 @@ function DateSelector({
     header: {
       fontSize: '12.47px',
       fontWeight: theme.typography.fontWeightMedium,
+      marginBottom: '8px',
```

Comment:
1 small tidying task:
- [ ] Remove the unused `label` prop in this file, line 50

1 aligning fix:
We can align the Tooltip that's next to the "Date Range" title by the following:

- [ ] Copy/paste into this files `const useStyle` the `iconStyle` CSS class from the "Request Type" located at the following.
loc: `/components/main/Desktop/TypeSelector/index.js` , line: 19

  - The `iconStyle={iconStyle}` is already being referenced in `ArrowToolTip` DOM element so no additional changes are necessary. Feel free to test on your local to check that the tooltip is vertically aligned with the title.

Reply:
done